example/api/servers.go

```diff
@@ -0,0 +1,23 @@
+// Package api provides a slice of Servers
+package api
+
+import (
+	"github.com/canonical/microcluster/rest"
+)
+
+// Servers represents the list of listeners that the daemon will start
+// Each Server has pre-defined endpoints that will be added to the listener
+// If the Server is marked as CoreAPI, its endpoints will be added to the core listener of Microcluster.
+var Servers = []rest.Server{
+	{
+		CoreAPI: true,
```

Comment:
What is the use case for having the `CoreAPI` field, if you can also define the endpoints as part of running `(m *MicroCluster) Start()` by providing the `extensionsAPI` slice?
 
Wouldn't both end up on the same listener?

Reply:
We are removing `extensionsAPI` with this PR. The idea is that we will use `extensionServers` for defining core API listener as well as additional listeners.

Reply:
Thanks, clear now. 

When adding this comment I haven't yet seen the commit that removes the `extensionsAPI` argument from the `Start()` func. The overall change wasn't fully clear to me by just reading the description of the PR :)